segment-event-notification/functions/track-sms.js

```diff
@@ -0,0 +1,25 @@
+exports.handler = async function(context, event, callback) {
+  const client = context.getTwilioClient();
+
+  if(event.type === 'track' && event.event === context.SEGMENT_EVENT) {
+    console.info(`Received a Segment track event named "${context.SEGMENT_EVENT}"`);
```

Comment:
should clear up the console statement

Reply:
This one was a bit of a request from our Segment contact; he wanted users to be able to view event flow through the integration via the Twilio debugger. Is there a better way to phrase the message or handle that part of the use case, or is it best left out?